tools.md

```diff
@@ -0,0 +1,206 @@
+---
+layout: page
+title: Our Tools
+description: The Tools we currently have at Hackland
+---
+
+<div class="tool">
+    <img class="tool-pic" src="/public/images/tools/garden.jpg"/>
+    <div class="tool-description">
+        <h3 class="tool-title">Garden</h3>
+        <p>
+            Can grow plants. Come and have fun trying to grow stuff to eat or not eat. 
+        </p>
+    </div>
+</div>
+
+<div class="tool">
+    <img class="tool-pic" src="/public/images/cnc.jpg"/>
+    <div class="tool-description">
+        <h3 class="tool-title">CNC Machine</h3>
+        <p>
+            Can cut full sized 2400x1200mm sheet materials, foams and other materials.
+        </p>
+        <p>
+            Fully dust extracted. Induction required to use!
+        </p>
+        <p>
+            Courses running for use (open to public) - register: <a href="https://www.meetup.com/HackLand/">https://www.meetup.com/HackLand/</a>
+        </p>
+    </div>
+</div>
+
+<div class="tool">
```

Comment:
I would suggest that all photos in this section be rotated so that their orientation is correct

Reply:
Sigh, just checked chrome. This isn't a problem in Firefox. There must be meta data in the images that only certain browsers use. Thanks!